### PR TITLE
CRM-19604: civicrm-ext-list: avoid default API limit of 25 results.

### PR DESCRIPTION
**Pull request by mlutfy**
* [CRM-19604: civicrm-ext-list: avoid default API limit of 25 results](https://issues.civicrm.org/jira/browse/CRM-19604)

---

 * [CRM-19404: Recommend PHP 5.3.23](https://issues.civicrm.org/jira/browse/CRM-19404)

### CODE REVIEW

File: drush/civicrm.drush.inc [hunk 1]
```diff
@@ -576,7 +576,7 @@ function civicrm_drush_help($section) {
 function drush_civicrm_ext_list() {
   civicrm_initialize();
   try{
-    $result = civicrm_api('extension', 'get', array('version' => 3));
+    $result = civicrm_api('extension', 'get', array('version' => 3, 'option.limit' => 0));
```
Comment (monishdeb) [inline drush/civicrm.drush.inc]: what if we use `civicrm_api3` instead as patch is against 7.x-master

Comment (seamuslee001) [reply drush/civicrm.drush.inc]: @monishdeb I think that makes sense to me and isn't the format 

``` php
array( 'version' => 3, options => array( 'limit' => 0 ))
```

**Comment by mlutfy** [comment]
Updated my PR as per feedback. Thanks for reviewing :)

**Comment by mlutfy** [comment]
hum the CRM reference in my commit is wrong. Replacing with PR #411.

**Pull request closed without merging**